[2023-10-01 05:52:33,278] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: our_third_dag_v3.first_task 2021-08-08T00:00:00+00:00 [queued]>
[2023-10-01 05:52:35,609] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: our_third_dag_v3.first_task 2021-08-08T00:00:00+00:00 [queued]>
[2023-10-01 05:52:35,755] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2023-10-01 05:52:35,778] {taskinstance.py:1043} INFO - Starting attempt 1 of 6
[2023-10-01 05:52:35,786] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2023-10-01 05:52:37,589] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): first_task> on 2021-08-08T00:00:00+00:00
[2023-10-01 05:52:38,081] {standard_task_runner.py:52} INFO - Started process 2287 to run task
[2023-10-01 05:52:38,458] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'our_third_dag_v3', 'first_task', '2021-08-08T00:00:00+00:00', '--job-id', '161', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/our_third_dag.py', '--cfg-path', '/tmp/tmpu1tyastb', '--error-file', '/tmp/tmpmmvu3md0']
[2023-10-01 05:52:38,974] {standard_task_runner.py:77} INFO - Job 161: Subtask first_task
[2023-10-01 05:52:45,944] {logging_mixin.py:104} INFO - Running <TaskInstance: our_third_dag_v3.first_task 2021-08-08T00:00:00+00:00 [running]> on host d4ed655215d1
[2023-10-01 05:52:58,857] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_third_dag_v3
AIRFLOW_CTX_TASK_ID=first_task
AIRFLOW_CTX_EXECUTION_DATE=2021-08-08T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-08-08T00:00:00+00:00
[2023-10-01 05:52:58,908] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2023-10-01 05:52:58,985] {bash.py:158} INFO - Running command: echo hello world, this is the first task!
[2023-10-01 05:53:02,665] {bash.py:169} INFO - Output:
[2023-10-01 05:53:04,287] {bash.py:173} INFO - hello world, this is the first task!
[2023-10-01 05:53:04,293] {bash.py:177} INFO - Command exited with return code 0
[2023-10-01 05:53:12,534] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=our_third_dag_v3, task_id=first_task, execution_date=20210808T000000, start_date=20231001T055233, end_date=20231001T055312
[2023-10-01 05:53:19,069] {local_task_job.py:188} WARNING - State of this instance has been externally set to success. Terminating instance.
[2023-10-01 05:53:19,552] {process_utils.py:100} INFO - Sending Signals.SIGTERM to GPID 2287
[2023-10-01 05:53:19,948] {taskinstance.py:1239} ERROR - Received SIGTERM. Terminating subprocesses.
[2023-10-01 05:53:19,989] {bash.py:185} INFO - Sending SIGTERM signal to bash process group
[2023-10-01 05:53:25,195] {process_utils.py:66} INFO - Process psutil.Process(pid=2287, status='terminated', exitcode=1, started='05:52:37') (2287) terminated with exit code 1
